test/onnx/test_utility_funs.py

```diff
@@ -260,7 +260,7 @@ def forward(self, x):
             assert node.kind() != "onnx::Concat"
             assert node.kind() != "onnx::Cast"
             assert node.kind() != "onnx::Constant"
-        assert len(list(graph.nodes())) == 2
+        assert len(list(graph.nodes())) == 1
```

Comment:
Add `assert node.kind() != "onnx::Add"` above as well?

Reply:
And node.kind() != onnx::Mul()/onnx::Div(), etc.?